### PR TITLE
change the style of three buttons

### PR DESCRIPTION
**Pull request by ChengshuLi**
@percyliang

### CODE REVIEW

File: codalab/apps/web/static/js/worksheet/bundle_uploader.jsx [hunk 1]
```diff
@@ -63,13 +63,20 @@ var BundleUploader = React.createClass({
     $(this.refs.fileDialog.getDOMNode()).trigger('click');
   },
   render: function () {
+    var upload_button = (
+      <Button
+        text='Upload'
```
Comment (percyliang) [inline codalab/apps/web/static/js/worksheet/bundle_uploader.jsx]: Since we have some space, call this 'Upload Bundle'

**Comment by percyliang** [comment]
Nice!
LGTM

**Comment by ChengshuLi** [comment]
@percyliang 

I change naming to 'Upload Bundle' and 'New Run Bundle'. I think 'New Run' is less clear about what it means as well. Then they may take up two lines instead of one line. But I think they look okay. What do you think? Are two lines of button acceptable? We may have more buttons in the future as well

**Comment by percyliang** [comment]
Let's actually roll back to 'Upload' and 'New Run'.

**Pull request merged**